app/Repositories/CategoryRepository.php

```diff
@@ -10,8 +10,22 @@ class CategoryRepository
 
     protected $model;
 
-    public function __construct(Category $category)
-    {
+    public function __construct(Category $category){
         $this->model = $category;
     }
+
+    public function createCategory($categoryNew){
+        $id = $this->model->create($categoryNew)->id;
+        return $id;
+    }
+
+    public function updateCategory($id, $categoryNew){
+        $this->model->where('id', $id)->update($categoryNew);
```

Comment:
Nếu cái `$this->model->where('id', $id)` = `NULL` nghĩa là ko có cái category nào tương ứng thì  `$this->model->where('id', $id)->update($categoryNew);` sẽ exception